.github/workflows/continuous-integration.yml

```diff
@@ -321,34 +321,9 @@ jobs:
         with:
           fetch-depth: 0
 
-      - name: Install dependencies
-        uses: ./.github/workflows/install
+      - name: Security Audit
         timeout-minutes: 30
-        with:
-          key: ${{ hashFiles('yarn.lock') }}
-          yarn_cache_folder: .cache/yarn
-          path: |
-            .cache/yarn
-            node_modules
-
-      - name: List changed files
-        id: changed-files
-        run: |
-          if [[ ${{ github.ref }} == 'refs/heads/main' ]]; then
-            echo ::set-output name=all_changed_files::$(git diff-tree --no-commit-id --name-only -r ${{ github.sha }})
-          else
-            echo ::set-output name=all_changed_files::$(git diff --name-only origin/main...)
-          fi
-
-      - name: Get app entries for changed files
-        id: get-changed-apps
-        run: echo ::set-output name=app_entries::$(node script/github-actions/get-changed-apps.js)
-        env:
-          CHANGED_FILE_PATHS: ${{ steps.changed-files.outputs.all_changed_files }}
```

Comment:
We'll probably want to keep these steps here and remove them from the [global action](https://github.com/department-of-veterans-affairs/platform-release-tools-actions/blob/main/security-check/action.yml#L22-L37) since they're specific to this repo